tests/test_closing.py

```diff
@@ -1569,6 +1569,34 @@ def try_pay():
     assert account_balance(l2, channel_id) == 0
 
 
+def test_listfunds_after_their_unilateral(node_factory, bitcoind):
+    """We keep spending info around for their unilateral closes.
+
+Make sure we the address.
+    """
+    coin_mvt_plugin = os.path.join(os.getcwd(), 'tests/plugins/coin_movements.py')
```

Comment:
```suggestion
    coin_mvt_plugin = os.path.join(os.path.dirname(__file__), 'plugins', 'coin_movements.py')
```

`os.path.join` will add the `/` as separator, having the elements contain `/` defeats the point (not all that useful since we only run on systems that have `/` as pat-separator, but nevertheless a good habit to have)

Reply:
Agreed.  I cut and paste from the previous, so I'll do a separate cleanup sweep PR.